frontend/docs/docs/deploy/customization.md

```diff
@@ -163,4 +163,6 @@ plausible.src = "https://plausible.io/js/script.js";
 plausible.defer = true;
 plausible.dataset.domain = "app.browsertrix.com";
 document.head.appendChild(plausible);
+
+window.analytics = window.plausible || function () { (window.plausible.q = window.plausible.q || []).push(arguments); };
```

Comment:
@emma-sg could we update the inject_analytics value to be `window.analytics` instead of `window.plausible`? Then we can keep the frontend tracking code generic, too.